pq-interop/pq-devnet-0.md

```diff
@@ -0,0 +1,126 @@
+# PQ Devnet-0 Specs
+
+
+### Devnet Change Log
+- No changes, initial devnet 
+
+
+### Active Discussions
+
+- **Sig Parameters:** Consensus on signature parameters (64 hash chains, chain length 8, max lifetime 2^18)? 
+- **Slot Duration:** 4s (Zeam) vs. 12s (Ream) – agree on one? 
+- **Networking Transport:** QUIC (Ream) vs. TCP first then QUIC (discussion in meeting)? 
+- **Validator Count:** Start with 1,000 keys (meeting consensus) and scale? 
+- **Consensus Mechanism:** Include minimal 3SF or no global consensus? 
+- **Infrastructure Setup:** Who handles (e.g., Grafana, Prometheus, Kurtosis-like for interop, Genesis Generator tool )? 
+
+
+## Sub-Spec List for PQ Devnet-0
+>The list below links to reference papers, specs, and implementations. Versions to be finalized by August 31, 2025.
+
+
+### Test Releases
+
+**Consensus Specs:** link to consensus spec release
+**Execution Spec Tests:** link to executuon spec release 
+
+### Spec Versions Required & Open PRs
+
+- List of links to spec PR's **Merged** :heavy_check_mark: 
+- List of links to spec PR's **Open** :exclamation:
+
+
+### Client Support Status
+>develop list of testing scenarios (link here)
+
+| Testing Scenario | Ream | Zeam | [TBD: Third Client?] |
+|------------------|------|------|----------------------|
+| 1a | :question: | :question: | :question: |
+| 2a | :question: | :question: | :question: |
+| 3a | :question: | :question: | :question: |
+| 4a | :question: | :question: | :question: |
+| 5a | :question: | :question: | :question: |
+| 6a | :question: | :question: | :question: |
+
+## Kurtosis Interop Config (Pre-Devnet Testing)
```

Comment:
do we have anyone working to fork kurtosis to support the lean ethereum?

Reply:
picked up comment in 'active discussions' section